### PR TITLE
check if device have access to network

### PR DESCRIPTION
**Pull request by porato-developer**
Check if the user device have access to network

### CODE REVIEW

**Comment by porato-developer** [comment]
HI @aliounekanoute i have already make changes according to your contribution.md file

File: lib/src/connectivity_bloc/connectivity_bloc.dart [hunk 1]
```diff
@@ -24,18 +24,29 @@ class ConnectivityBloc extends Bloc<ConnectivityEvent, ConnectivityState> {
...
+        .listen((ConnectivityResult result) async {
+      if (result != ConnectivityResult.none) {
+        // If the connectivity result is none , verify if device really have access to network.
+        try
+        {
+          final response = await http.get(Uri.parse('https://www.google.com'));
+          if(response.statusCode==200)
+          {
+            add(OnConnectivityEvent());
+          }
```
Comment (aliounekanoute) [inline lib/src/connectivity_bloc/connectivity_bloc.dart:36]: What happens if the response status is not 200?

Comment (porato-developer) [reply lib/src/connectivity_bloc/connectivity_bloc.dart:36]: I started from the principle that if he has the connection the code must be 200 and if not we will inevitably have an exception that i catch below

Comment (porato-developer) [reply lib/src/connectivity_bloc/connectivity_bloc.dart:36]: But if you want i can add an **else** clause

Comment (aliounekanoute) [reply lib/src/connectivity_bloc/connectivity_bloc.dart:36]: If it is sure that the SocketException is thrown when the response status is different from 200, it is ok for me.

Comment (porato-developer) [reply lib/src/connectivity_bloc/connectivity_bloc.dart:36]: yes, you can check it on your side before validation the PR

**Review by aliounekanoute — APPROVED** [review]
Good job, thanks a lot 🙏🏾.

**Pull request merged**